src/apps/reservation-list/modal/reservation-details/reservation-details.test.ts

```diff
@@ -466,6 +483,23 @@ describe("Reservation details modal test", () => {
     cy.getBySel("reservation-form-button", true).click();
     cy.getBySel("modal-cta-button", true).click();
 
+    cy.wait("@put-library-branch-and-expiry-date").should(({ request }) => {
+      expect(JSON.stringify(request.body)).to.equal(
+        JSON.stringify({
+          reservations: [
+            {
+              expiryDate: "2022-09-21",
+              // TODO: This should be DK-775120 as this is what the user
+              //  selected in previous steps but that does not seem to work
+              //  right now.
+              pickupBranch: "DK-775160",
+              reservationId: 4698559133
+            }
```

Comment:
As I see it i have identified a bug here. The reservation sent to FBS does not actually use the updated value. That should be handled in a separate issue though.

Reply:
Great. Did you already create a task for it?